### ISSUE TITLE
Format of DDNS_HOOK_KEY_FILE

### ISSUE THREAD

**Issue by haf** [eferdman/dnspython-hook#12]
What format should it have? INI?

**Comment by umlaeute** [comment]
that's a standard ISC-file, as used by `bind` to setup the rndc key.

E.g. the key-config in the [example configuration](https://github.com/eferdman/dehydrated-hook-ddns-tsig/blob/master/dehydrated-hook-ddns-tsig.conf) would look like:

~~~ISC
key "testkey" {
   secret "R3HI8P6BKw9ZwXwN3VZKuQ==";
   algorithm = hmac-md5;
};
~~~

this probably ought to be added to the README

**Comment by umlaeute** [comment]
for what it worth, the format *is* already documented in the [README](https://github.com/eferdman/dehydrated-hook-ddns-tsig/blob/master/README.md#using-an-extra-key-file), by means of an external link.

**Comment by umlaeute** [comment]
@eferdman this ticket can now be closed as well (with 5a0aa57).

Could you also please tag a new release?

**Comment by haf** [comment]
Thank you for your reply
